src/utils/error-to-string.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function errorToString(err) {
+  const isString = typeof error === 'string';
+  const hasMessage = err != null && typeof err.msg === 'string';
+  if (isString) {
+    return err;
+  } else if (hasMessage) {
+    return err.msg;
+  }
+
+  return err + '';
```

Comment:
what about `err.toString()` which perhaps is a bit more readable

Reply:
Hmm... That might cause errors like `TypeError: Cannot read property 'toString' of undefined` if it is null or undefined.

Reply:
good point!